ts/src/scenario.ts

```diff
@@ -213,10 +218,7 @@ export class Scenario {
    */
   async addPlayerWithApp(appWithOptions: AppWithOptions) {
     await this.ensureLocalServices();
-    const conductor = await this.addConductor();
-    if (appWithOptions.options?.networkConfig) {
-      conductor.setNetworkConfig(appWithOptions.options.networkConfig);
-    }
+    const conductor = await this.addConductor(appWithOptions.options?.networkConfig);
```

Comment:
Previously we were launching the conductor, then overriding the network settings. Thus the network settings were not actually applied.

Now we override the network settings before launching the conductor.